src/new-client/src/app/lib/types/index.tsx

```diff
@@ -211,10 +211,10 @@ type ClinicalService = {
 };
 
 enum AppointmentStatus {
-  Unknown,
-  Provisional,
-  Booked,
-  Cancelled,
+  Unknown = 'Unknown',
+  Provisional = 'Provisional',
+  Booked = 'Booked',
+  Cancelled = 'Cancelled',
```

Comment:
Are these strings needed because otherwise `AppointmentStatus.Provisional` is coming out as `1`? I'd argue at this point it'd be safer to just convert it to a union type, then there's chance for confusion or differences in compiler/runtime behaviour. 

If it stays as an enum, wouldn't the same strings be required for `ContactItemType`?